src/components/navbar/Wallet.tsx

```diff
@@ -69,28 +68,33 @@ export default function () {
           <hr className={delimeterContainer} />
         </>
       )}
-      <div
-        className={accountLinkContainer}
-        onClick={async () => {
-          if (!account) return await WalletStore.connect(true)
-          openNewTabAndFocus(getEtherscanAddressUrl(account, Network.Goerli))
-        }}
+      <a
+        href={
+          account ? getEtherscanAddressUrl(account, Network.Goerli) : undefined
```

Comment:
Don't we need to place it inside `div.accountLinkContainer`?

Reply:
This will break the layout